website/docs/admin_settings_system.md

```diff
@@ -13,18 +13,44 @@ Settings applicable to the full studio.
 
 ![general_settings](assets/settings/settings_system_general.png)
 
-**`Studio Name`** - Full name of the studio (can be used as variable on some places)
+### Studio Name
+ - Full name of the studio (can be used as variable on some places)
 
-**`Studio Code`** - Studio acronym or a short code (can be used as variable on some places)
+### Studio Code
+ - Studio acronym or a short code (can be used as variable on some places)
 
-**`Admin Password`** - After setting admin password, normal user won't have access to OpenPype settings
+### Admin Password
+ - After setting admin password, normal user won't have access to OpenPype settings
 and Project Manager GUI. Please keep in mind that this is a studio wide password and it is meant purely
 as a simple barrier to prevent artists from accidental setting changes.
 
-**`Environment`** - Globally applied environment variables that will be appended to any OpenPype process in the studio.
-
-**`Disk mapping`** - Platform dependent configuration for mapping of virtual disk(s) on an artist's OpenPype machines before OP starts up.
-Uses `subst` command, if configured volume character in `Destination` field already exists, no re-mapping is done for that character(volume).
+### Environment
+ - Globally applied environment variables that will be appended to any OpenPype process in the studio.
+ - OpenPype is using some keys to configure some tools. Here are some:
+
+#### OPENPYPE_TMPDIR:
+ - Custom staging dir directory
+ - Supports anatomy keys formating. ex `{root[work]}/{project[name]}/temp`
+ - supported formating keys:
+    - root[work]
+    - project[name | code]
+    - asset
+    - hierarchy
+    - task
+    - username
+    - app
+
+#### OPENPYPE_DEBUG
+ - setting logger to debug mode
+ - example value: "1" (to activate)
+
+#### OPENPYPE_LOG_LEVEL
+ - stringified numeric value of log level. [Here for more info](https://docs.python.org/3/library/logging.html#logging-levels)
+ - example value: "10"
```

Comment:
I would actually revert this docs change and move the special environment variables documentation into a page of its own. So that in Admin section for Configuration there's an "Environment Variables" page. This particular area of the docs could then just link to that page.

Why separate it? Because I'd actually argue that most of these environment variables mentioned here aren't supposed to be set in the global **environment** under system settings. But instead a particular machine could have a `OPENPYPE_LOG_LEVEL` env locally, or even have a "Environments" override in "Local Settings". There are many places where these are more likely to be used.

I'd just make a page dedicated for environment variables that are there to customize OP behavior. I'd also add `OPENPYPE_MONGO`, `OPENPYPE_USERNAME`, `OPENPYPE_LOCAL_ID`. Maybe even categorize some under **advanced** on that page.

Same goes for documenting `OPENPYPE_FFMPEG_PATHS` and `OPENPYPE_FFMPEG_PATHS`.

Reply:
Hey @BigRoy thanks for the suggestion. I have added this to backlog https://github.com/ynput/OpenPype/issues/4442

Reply:
I have implemented the comment partly at this PR - I really liked the suggestion for separated Environment section. But I have added it under `Admin/Configuration`

Reply:
@BigRoy If you are having suggestion for the Section introduction text, I am very happy to add it in it ;)